webpack/frontendModules.config.js

```diff
@@ -35,6 +35,9 @@ const {
 
 module.exports = ( mode ) => ( {
 	entry: {
+		// Consent Mode
+		'consent-mode': './assets/js/consent-mode/consent-mode.js',
```

Comment:
Noting here that the source file does not need to be prefixed but the URL reference to the built output in `dist/assets/js` would be.

```suggestion
		'googlesitekit-consent-mode': './assets/js/consent-mode/consent-mode.js',
```